EIPS/eip-7069.md

```diff
@@ -50,31 +50,32 @@ Lastly, the introduction of the `RETURNDATA*` instructions ([EIP-211](./eip-211.
 
 We introduce four new instructions:
 
-- `CALL2` (`0xf8`) with arguments `(target_address, input_offset, input_size, value)` 
-- `DELEGATECALL2` (`0xf9`) with arguments `(target_address, input_offset, input_size)`
-- `STATICCALL2` (`0xfb`) with arguments `(target_address, input_offset, input_size)`
+- `EXTCALL` (`0xf8`) with arguments `(target_address, input_offset, input_size, value)` 
+- `EXTDELEGATECALL` (`0xf9`) with arguments `(target_address, input_offset, input_size)`
+- `EXTSTATICCALL` (`0xfb`) with arguments `(target_address, input_offset, input_size)`
 - `RETURNDATALOAD` (`0xf7`) with argument `offset`
 
-Execution semantics of `*CALL2`:
+Execution semantics of `EXT*CALL`:
 
 1. Charge `WARM_STORAGE_READ_COST` (100) gas.
-2. Pop required arguments from stack, fail with error on stack underflow.
-4. If `value` is non-zero:
-    3a. Fail with error if the current frame is in `static-mode`.
-    3b. Fail with error if the balance of the current account is less than `value`.
-    3c. Charge `CALL_VALUE_COST` gas.
+2. Pop required arguments from stack, halt with exceptional failure on stack underflow.
+3. If `value` is non-zero:
+    - Halt with exceptional failure if the current frame is in `static-mode`.
+    - Charge `CALL_VALUE_COST` gas.
 4. Peform (and charge for) memory expansion using `[input_offset, input_size]`.
 5. If `target_address` is not in the `warm_account_list`, charge `COLD_ACCOUNT_ACCESS - WARM_STORAGE_READ_COST` (2500) gas.
 6. If `target_address` is not in the state and the call configuration would result in account creation, charge `ACCOUNT_CREATION_COST` (25000) gas.
     - The only such case in this EIP is if `value` is non-zero.
-7. Calculate the gas available to callee as caller's remaining gas reduced by `max(ceil(gas/64), MIN_RETAINED_GAS)` (`MIN_RETAINED_GAS` is 5000).
-8. Fail with error if the gas available to callee at this point is less than `MIN_CALLEE_GAS` (2300).
-9. Perform the call with the available gas and configuration.
-10. Push a status code on the stack:
-    11a. `0` if the call was successful.
-    11b. `1` if the call has reverted.
-    11c. `2` if the call has failed.
-11. Gas not used by the callee is returned to the caller.
+7. Calculate the gas available to callee as caller's remaining gas reduced by `max(floor(gas/64), MIN_RETAINED_GAS)`.
+8. Halt with exceptional failure if the gas available to callee at this point is less than `MIN_CALLEE_GAS`.
```

Comment:
Why is this exceptional failure? Seems could be light failure

Reply:
The PR only changes the language on this one. And I see it as a sensible failure mode - consistent with an OOG. But light failure also makes sense on 2nd thought. @chfast @axic @shemnon any more opinions on this, any objections to changing to light failure?